plugins/history_plugin/history_plugin.cpp

```diff
@@ -44,14 +44,14 @@ namespace eosio {
 
    struct by_action_sequence_num;
    struct by_account_action_seq;
-   struct by_trx_id;
+   struct by_trx_id_act_seq;
```

Comment:
Can we rename these? Are the typenames stored in shared memory?

Reply:
We can't rename these. Please revert.

Reply:
chainbase in develop is already incompatible with 2.0.x.

Reply:
Ah, good point.

Reply:
So the verdict is we can?  I had renamed them because of compiler errors after some new includes.

Reply:
Yes. This is fine. Users are going to have to re-create their shared mem anyway.